judgments/templatetags/document_utils.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def get_title_to_display_in_html(document_title, document_type):
```

Comment:
filter so we remove the "Press Summary of " prefix where we need to, but still keep the title of theml including it so we see it in the tab of a browser. Did consider passing through two context variables, one with stripped prefix but I thought this was cleaner.